src/styles/annotator/notebook.scss

```diff
@@ -24,4 +24,23 @@
   padding: 0;
   width: 100%;
   height: 100%;
+  border: none;
+}
+
+.Notebook__controller-bar {
+  position: absolute;
+  width: calc(100vw - #{var.$layout-space * 2});
+  display: flex;
+  justify-content: flex-end;
+  background-color: var.$grey-2;
+}
+
+.Notebook__close-button {
+  position: absolute;
+  @include buttons.button--labeled(
```

Comment:
Ah! I presume this is to adapt for not being on white background? 👍🏻 I will capture this, as I'd like to bake this kind of adaptation into the new `Button*` work.

Reply:
Correct.